local/running_environment.ex

```diff
@@ -93,14 +95,23 @@ defmodule AntikytheraLocal.RunningEnvironment do
   defp fetch_current_version(app_name_str) do
     url = "http://#{AntikytheraCore.Cmd.hostname()}:8080/versions"
     token = File.read!(@system_info_access_token_path)
-    %Httpc.Response{status: 200, body: body} = Httpc.get!(url, %{"authorization" => token})
+    Httpc.get(url, %{"authorization" => token})
+    |> R.bind(fn %Httpc.Response{status: 200, body: body} -> extract_version_str(body, app_name_str) end)
+  end
+
+  defp extract_version_str(body, app_name_str) do
     String.split(body, "\n", trim: true)
-    |> Enum.find_value(fn line ->
-      case String.split(line) do
-        [^app_name_str, v] -> v
-        _                  -> nil
-      end
-    end)
+    |> Enum.find(fn line -> String.starts_with?(line, app_name_str) end)
```

Comment:
This could lead to subtle bugs.
Consider the case where `app_name_str` is `"foobar"` and there's a line `"foobarbaz 1.2.3"` in the body.
The problem here is that we must always take delimiters into consideration; only looking at each element is (most of the time) not sufficient.
(And this is one of the most crucial point in string manipulation!)
In this case, `String.starts_with?(line, "#{app_name_str} ")` is OK.

Another minor issue is that we inspect the matched line again at l.110.
It'd be better if we could unite these line manipulations.

So, my recommendation is to define `Antikythera.EnumUtil.find_value!/2`.
(we don't need detailed error message here)
```ex
String.split(body, "\n", trim: true)
|> EnumUtil.find_value!(fn line ->
  case String.split(line) do
    [^app_name_str, v] -> v
    _                  -> nil
  end
end)
```

Reply:
Sorry for misunderstanding your comments.
Implement `Antikythera.EnumUtil.{find!/2, find_value!/2}` in 1509b33,
then I fixed to use it in 3632bda.

Does my implementation match what you expect?

Reply:
Yes, now it looks nicer!